sass/coveo-styleguide/scss/controls/_checkboxes.scss

```diff
@@ -93,9 +93,7 @@ input[type='checkbox'].coveo-checkbox {
   }
 
   &:disabled + button {
-    background-color: $dark-grey;
```

Comment:
Hmm, I'm curious why this line did not play well with the IEditor. Is it not needed in the search-ui?